### PR TITLE
feat(argo-cd): Add affinity to redisSecretInit Job

### PR DESCRIPTION
**Pull request by killianmuldoon**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Comment by tico24** [comment]
Thanks for your PR. 

If you could please update the title and complete the changelog, then we can get to reviewing.

**Comment by tico24** [comment]
Or @jmeridth could do it for you just to make me look grumpy...

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -1725,6 +1725,9 @@ redisSecretInit:
   # @default -- `""` (defaults to global.priorityClassName)
   priorityClassName: ""
 
+  # -- Assign custom [affinity] rules to the Redis secret-init Job
+  affinity: |
```
Comment (tico24) [inline charts/argo-cd/values.yaml:1729]: Everywhere else in the values file, affinity is defined as 

```
affinity: {}
```

So this should probably do the same for consistency.

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ appVersion: v2.12.4
 kubeVersion: ">=1.25.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 7.6.7
+version: 7.6.8
 home: https://github.com/argoproj/argo-helm
 icon: https://argo-cd.readthedocs.io/en/stable/assets/logo.png
 sources:
```
Comment (tico24) [inline charts/argo-cd/Chart.yaml:9]: requires changelog

**Review by tico24 — CHANGES_REQUESTED** [review]

**Comment by killianmuldoon** [comment]
Thanks for the quick review - I believe it should be in line now.

**Review by tico24 — APPROVED** [review]

**Review by yu-croco — APPROVED** [review]

**Pull request merged**